bed/compare.go

```diff
@@ -68,6 +116,39 @@ func Compare(a Bed, b Bed) int {
 	return 0
 }
 
+//CompareSize returns zero for beds with an equal length (ChromEnd - ChromStart) and otherwise returns the ordering of the two Bed structs.
+func CompareSize(a Bed, b Bed) int {
+	sizeA := a.ChromEnd - a.ChromStart
+	sizeB := b.ChromEnd - b.ChromStart
+	if sizeA < sizeB {
+		return -1
+	}
+	if sizeA > sizeB {
+		return 1
+	}
+	return 0
+}
+
+//Compare returns zero for beds with an equal ChromEnd position and otherwise returns the ordering of the two Bed entries by chromEnd.
+func CompareChromEnd(a Bed, b Bed) int {
```

Comment:
I think we should remove many of these compare functions. Or at least make them private. I would strongly argue that `if a.ChromEnd < b.ChromEnd {...}` is always clearer then `if CompareChromEnd(a, b) == -1 {...}`